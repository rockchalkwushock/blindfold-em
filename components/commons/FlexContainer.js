const FlexContainer = ({ children }) => (
  <div>
    {children}
    <style jsx>{`
      div {
        display: flex;
        flex-direction: column;
        margin: auto;
        max-width: 50em;
        min-height: 100vh;
      }
    `}</style>
  </div>
)

export default FlexContainer
