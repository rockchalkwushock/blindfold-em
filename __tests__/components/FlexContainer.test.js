import React from 'react'
import { render } from 'enzyme'

import FlexContainer from '../../components/commons/FlexContainer'

const Test = () => (
  <FlexContainer>
    <div>1</div>
    <div>2</div>
    <div>3</div>
  </FlexContainer>
)

describe('Component: <FlexContainer />', () => {
  test('should render without exploding', () => {
    const tree = render(<Test />)
    expect(tree).toMatchSnapshot()
  })
})
