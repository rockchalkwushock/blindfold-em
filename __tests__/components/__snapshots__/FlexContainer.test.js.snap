// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <FlexContainer /> should render without exploding 1`] = `
<div
  class="jsx-1111547334"
>
  <div>
    1
  </div>
  <div>
    2
  </div>
  <div>
    3
  </div>
</div>
`;
